sql/2020/19_Compression/party_of_content_encoding.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+#party of content encoding
+SELECT
+  client,
+  CASE
+   WHEN req_host!= ' ' THEN 'first-party'
+   ELSE 'thrid-party' END
+  AS party,
```

Comment:
I think first and third parties should be based on the `third_parties` data:

```suggestion
  IF(NET.HOST(url) IN (
    SELECT
      domain
    FROM
      `httparchive.almanac.third_parties`
    WHERE
      date = '2020-08-01' AND
      category != 'hosting'
  ), 'third party', 'first party') AS party
```

Reply:
thanks to the clarification.